pkg/services/ngalert/schedule/schedule.go

```diff
@@ -424,84 +422,110 @@ func (sch *schedule) ruleEvaluationLoop(ctx context.Context) error {
 }
 
 func (sch *schedule) ruleRoutine(grafanaCtx context.Context, key models.AlertRuleKey, evalCh <-chan *evalContext, stopCh <-chan struct{}) error {
-	sch.log.Debug("alert rule routine started", "key", key)
+	logger := sch.log.New("uid", key.UID, "org", key.OrgID)
```

Comment:
I believe that the logger is threadsafe, what's the reason for creating a new logger here?